akka-projection-core/src/main/scala/akka/projection/Projection.scala

```diff
@@ -90,6 +100,8 @@ private[projection] object RunningProjection {
         .andThen {
           case Success(_) =>
             statusObserver.stopped(projectionId)
+          case Failure(AbortProjectionException) =>
+            statusObserver.stopped(projectionId) // no restart
           case Failure(exc) =>
             Try(statusObserver.stopped(projectionId))
             statusObserver.restarted(projectionId, exc)
```

Comment:
Unrelated to this PR, but I think we should not call it `restarted`, but `failed`. 

The only thing we know here is that this stream stopped because of a failure. We can't no if it will restart. For instance, a user may have configured (via settings) the backoff for a limited number of retries. 

That's a detail maybe, but if we call it `restarted` and we keep using that term, we may give the wrong impression.